jaxrs-tck/docs/JAXRSTCK4.0-ReleaseNotes.html

```diff
@@ -56,66 +45,66 @@ <h2>Kit Contents </h2>
           <li>
             <p>A <strong>signature test</strong> that checks that all of the
               public APIs are supported or defined in the Jakarta EE RESTful Web
-              Services 3.1 implementation that is under test.</p>
+              Services 4.0 implementation that is under test.</p>
           </li>
           <li>
             <p><strong>Specification tests</strong> for the testable assertions
-              required by the Jakarta EE RESTful Web Services 3.1 Specification.</p>
+              required by the Jakarta EE RESTful Web Services 4.0 Specification.</p>
           </li>
           <li>
             <p><strong>API tests</strong> for the public APIs and language
-              elements defined by the Jakarta EE RESTful Web Services 3.1
+              elements defined by the Jakarta EE RESTful Web Services 4.0
               Specification.</p>
           </li>
         </ul>
       </li>
     </ul>
     <hr>
-    <h2>Jakarta EE RESTful Web Services 3.1 TCK Facts</h2>
+    <h2>Jakarta EE RESTful Web Services 4.0 TCK Facts</h2>
     <p>The test suite bundle contains the following:</p>
     <ul>
-      <li>2796 total tests</li>
+      <li>2722 total tests out of which 128 tests are excluded</li>
     </ul>
     <hr>
     <h2>Platform Notes</h2>
-    <p>The Jakarta EE RESTful Web Services 3.1 TCK tests and the Jakarta EE
-      RESTful Web Services 3.1 Compatible Implementation (CI) can be run with
-      Java Platform, Standard Edition version 11.0 (Java SE 11). 
+    <p>The Jakarta EE RESTful Web Services 4.0 TCK tests and the Jakarta EE
+      RESTful Web Services 4.0 Compatible Implementation (CI) can be run with
+      Java Platform, Standard Edition version 17.0 (Java SE 17). 
```

Comment:
My understanding was that it would need to be run on both 17 and 21.   Did I misunderstand?